packages/0x.js/test/0x.js_test.ts

```diff
@@ -137,6 +135,7 @@ describe('ZeroEx library', () => {
             const proxyAddress = zeroEx.proxy.getContractAddress();
             const txHash = await zeroEx.token.setUnlimitedProxyAllowanceAsync(zrxTokenAddress, coinbase);
             const txReceiptWithDecodedLogs = await zeroEx.awaitTransactionMinedAsync(txHash);
+            // tslint:disable-next-line:no-unnecessary-type-assertion
```

Comment:
I think this might be necessary... It looks like a TSLint bug where it isn't taking into account the typeVariable when doing type equality. It therefore thinks we are casting to it's existing type, but we are actually casting to the same type, with a different typeVariable. In this specific example: `LogWithDecodedArgs<DecodedLogArgs>` is being cast to `LogWithDecodedArgs<ApprovalContractEventArgs>`.